docs/src/extend/plugins.md

```diff
@@ -104,24 +110,32 @@ module.exports = {
             }
         }
     }
+    rules: {
+        "my-rule": {/* rule definition */},
+        "another-rule": {/* rule definition */},
+        "yet-another-rule": {/* rule definition */},
+    }
 };
 ```
 
-If the example plugin above were called `eslint-plugin-myPlugin`, the `myConfig` and `myOtherConfig` configurations would then be usable by extending off of `"plugin:myPlugin/myConfig"` and `"plugin:myPlugin/myOtherConfig"`, respectively.
+Plugins cannot force a specific configuration to be used. Users must manually include plugin's configurations in their configuration file.
+
+If the example plugin above were called `eslint-plugin-myPlugin`, the `myConfig` and `myOtherConfig` configurations would then be usable in a configuration file by extending `"plugin:myPlugin/myConfig"` and `"plugin:myPlugin/myOtherConfig"`, respectively.
 
 ```json
+// eslintrc.json
+
 {
     "extends": ["plugin:myPlugin/myConfig"]
 }
 
 ```
 
-**Note:** Please note that configuration will not enable any of the plugin's rules by default, and instead should be treated as a standalone config. This means that you must specify your plugin name in the `plugins` array as well as any rules you want to enable that are part of the plugin. Any plugin rules must be prefixed with the short or long plugin name. See [Configure Plugins](../use/configure/plugins#configure-plugins) for more information.
-
 ### Peer Dependency
 
-To make clear that the plugin requires ESLint to work correctly you have to declare ESLint as a `peerDependency` in your `package.json`.
-The plugin support was introduced in ESLint version `0.8.0`. Ensure the `peerDependency` points to ESLint `0.8.0` or later.
+To make clear that the plugin requires ESLint to work correctly, you must declare ESLint as a `peerDependency` in your plugin's `package.json`.
+
+Plugin support was introduced in ESLint version `0.8.0`. Ensure the `peerDependency` points to ESLint `0.8.0` or later.
```

Comment:
The official name of the key is package.json is [`peerDependencies` ](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#peerdependencies). We should use this to avoid any confusion. What do you think? 

```suggestion
To make clear that the plugin requires ESLint to work correctly, you must declare ESLint as a peer dependency by mentioning it in the `peerDependencies` field of your plugin's `package.json`.

Plugin support was introduced in ESLint version `0.8.0`. Ensure the `peerDependencies` points to ESLint `0.8.0` or later.
```